plugins/k8saudit-gke/pkg/k8sauditgke/config.go

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sauditgke
+
+import (
+	"log"
+
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk"
+	"github.com/falcosecurity/plugins/plugins/k8saudit/pkg/k8saudit"
+	"github.com/patrickmn/go-cache"
+	"google.golang.org/api/container/v1"
+)
+
+type Plugin struct {
+	k8saudit.Plugin
+	logger *log.Logger
```

Comment:
I see you point to create a logger, but Falco shoud be in charge to print the lines in the stdout, to be sure to have a consistency of the format

Reply:
I guess I must be missing something, but how do you write to the falco logger from the plugin sdk?

Both the [k8saudit](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/pkg/k8saudit/k8saudit.go#L44) and [k8saudit-eks](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit-eks/pkg/k8sauditeks/k8sauditeks.go#L47) are creating a plugin logger as well.

The `NewPushInstance` method from the sdk only takes a go channel to pass `PushEvent`s to falco. And the `Plugin` interface does not seem to offer a logger?

Reply:
You're right, I forgot that. We should see in the future to extend to API/SDK in Falco for that